src/actions/propertiesActions.js

```diff
@@ -0,0 +1,34 @@
+// Create Redux action types
+export const GET_PROPERTIES = 'GET PROPERTIES'
+export const GET_PROPERTIES_SUCCESS = 'GET_PROPERTIES_SUCCESS'
+export const GET_PROPERTIES_FAILURE = 'GET_PROPERTIES_FAILURE'
+
+// Create Redux action creators that return an action
+export const getProperties = () => ({
+  type: GET_PROPERTIES,
+})
+
+export const getPropertiesSuccess = properties => ({
+  type: GET_PROPERTIES_SUCCESS,
+  payload: properties,
+})
+
+export const getPropertiesFailure = () => ({
+  type: GET_PROPERTIES_FAILURE,
+})
+
+// Combine them all in an asynchronous thunk
+export function fetchProperties() {
+  return async dispatch => {
+    dispatch(getProperties())
+
+    try {
+      const response = await fetch('http://www.solideveloper.com/api/property/')
+      const data = await response.json()
```

Comment:
Here I'm making the call. Ideally you are going to have an actions page for each model of the API (eg experience, bookings, etc)